tests/test_project.py

```diff
@@ -2056,41 +2056,6 @@ def setup_main_interface(self, request):
         os.chdir(self._tmp_dir.name)
         request.addfinalizer(self.switch_to_cwd)
 
-
-class TestAggregatesProjectUtilities(TestAggregatesProjectBase):
-    def test_AggregatesCursor(self):
-        project = self.mock_project()
-        agg_cursor = _AggregateStoresCursor(project=project)
-        # All operations will return aggregates, even if the aggregates are not
-        # unique to that operation, because every operation/group in this
-        # project has a custom aggregator defined. Only the default aggregator
-        # can de-duplicate the returned results in the cursor, thus it is
-        # expected that the length of the cursor is larger than the number of
-        # unique ids present in the cursor.
-        assert len(agg_cursor) == 42
-        assert len({get_aggregate_id(agg) for agg in agg_cursor}) == 34
-        assert tuple(project) in agg_cursor
-        assert all((job,) in agg_cursor for job in project)
```

Comment:
Has all the behavior being tested in this test been replicated in the new test class?

Reply:
Yes, I think @b-butler tested the core functionality correctly. I pushed a commit to further expand coverage: 17ff07f9a8a917eb19b5c957eb9129e1073c1007